android/src/main/kotlin/de/mintware/barcode_scan/BarcodeScannerActivity.kt

```diff
@@ -17,7 +17,6 @@ class BarcodeScannerActivity : Activity(), ZXingScannerView.ResultHandler {
 
     private lateinit var config: Protos.Configuration
     lateinit var scannerView: ZXingScannerView
```

Comment:
Maybe it's better to change the `scannerView` to a nullable private var (just to prevent further problems like this one):
```kt
private var scannerView: ZXingScannerView? = null
```